src/op_mode/conntrack_sync.py

```diff
@@ -151,35 +143,53 @@ def xml_to_stdout(xml):
             call(f'sudo {conntrackd_bin} -C {conntrackd_config} -s exp')
```

Comment:
As mentioned above, `sudo` should be consistent across the whole script, if we rewrite it.